cypress/e2e/5-code-node.cy.ts

```diff
@@ -0,0 +1,36 @@
+import { WorkflowsPage as WorkflowsPageClass } from '../pages/workflows';
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const WorkflowsPage = new WorkflowsPageClass();
+const WorkflowPage = new WorkflowPageClass();
+
+describe('Code node', () => {
+	beforeEach(() => {
+		cy.task('db:reset');
+		cy.skipSetup();
+	});
+
+	it('should execute the placeholder in all-items mode successfully', () => {
+		WorkflowsPage.actions.createWorkflowFromCard();
+		WorkflowPage.actions.addInitialNodeToCanvas('Manual Trigger');
+		WorkflowPage.actions.addNodeToCanvas('Code');
+		WorkflowPage.actions.openNodeNdv('Code');
+
+		WorkflowPage.actions.executeNodeFromNdv();
+
+		WorkflowPage.getters.successToast().contains('Node executed successfully');
+	});
+
+	it('should execute the placeholder in each-item mode successfully', () => {
+		WorkflowsPage.actions.createWorkflowFromCard();
```

Comment:
This is a nitpick, but instead of using two page objects (`/workflows` and `/workflow/new`) you can go straight to the new workflow page.
I have created an [action for this](https://github.com/n8n-io/n8n/blob/66175e06dd7aeb72de1e11e815932586c8adffb1/cypress/pages/workflow.ts#L49) (which also handles loaders) but it's not merged in yet.